api/src/pcapi/routes/backoffice/venues/forms.py

```diff
@@ -189,7 +189,7 @@ def validate_all_permanent(self, all_permanent: fields.PCCheckboxField) -> field
 
 
 class RemovePricingPointForm(utils.PCForm):
-    comment = fields.PCCommentField("Commentaire interne")
+    comment = fields.PCCommentField("Commentaire qui apparaîtra sur le lieu")
```

Comment:
En fait non, car ici tu modifies le nom pour la suppression du point de valorisation, alors que c'est dans ce cas effectivement un commentaire interne.

Il faut afficher "Commentaire qui apparaîtra sur le lieu" uniquement pour le cas où le commentaire sera affiché sur le lieu dans PC Pro, donc lorsque `venue.comment` est rempli : la suppression de SIRET.

Donc deux choix : 
- soit enlever l'héritage de classe, et définir les deux champs dans `RemoveSiretForm`
- soit changer le `self.comment.label` dans `RemoveSiretForm.__init__` (à tester)

Reply:
Alors en soit j'ai compris ce que tu me demande d'un point de vue technique cependant, j'essaye de le tester d'un point de vue métier et j'ai beaucoup de mal. Le lien entre venue et offerer étant partiellement "virtuelle " et je ne vois pas dans la db de test d'offerer lié a plusieurs venue, je ne vois pas comment le voir/modifier en sql car le lien n'existe pas vraiment en sql mais uniquement en sqlalchemy. ( il faut que ce soit lié a plusieurs venue afin que l'on puisse avoir la possibilité de supprimer un siret ) 

Je suis d'autant plus confus que je vois que tu as approuvé malgré ta remarque le ticket... j'ose plus toucher a rien du coup car je ne veux pas modifier la mauvaise chose et je ne veux pas accepté si ce n'est pas vraiment approuvé

Reply:
Regarde bien l'historique ci-dessous, j'ai poussé la correction hier soir, pour que ce ticket trivial passe dans l'itération pour la MEP de lundi.

Pour tester, rien de plus simple, il y a dans la sandbox une « structure avec plusieurs lieux » qui se trouve facilement. Sinon, tu pouvais aussi créer la structure et les lieux à partir de PC Pro en local.